pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -392,16 +392,16 @@ func GenerateCurrentInstallStrategy(config *operatorutil.KubeVirtDeploymentConfi
 	rbaclist = append(rbaclist, rbac.GetAllController(config.GetNamespace())...)
 	rbaclist = append(rbaclist, rbac.GetAllHandler(config.GetNamespace())...)
 
+	monitorServiceAccount := config.GetMonitorServiceAccount()
 	if monitorNamespace != "" {
```

Comment:
This is also misleading.. Not that it's your fault, but it's not clear that this means that the service monitor was not found in any namespace.

I would consider maybe:
```go
isServiceMonitorFound := monitorNamespace != ""
if isServiceMonitorFound {
...
```

A refactoring can definitely make this code clearer, but perhaps it's better to do that in future PRs.

Reply:
I guess you refereed to the ServiceAccount and not to the serviceMonitor,
I Agree this is misleading :) and this is why i modified the warning massage .